src/uag.c

```diff
@@ -906,6 +906,10 @@ struct ua *uag_find_msg(const struct sip_msg *msg)
 	cuser = &msg->uri.user;
 	for (le = uag.ual.head; le; le = le->next) {
 		struct ua *ua = le->data;
+		struct account *acc = ua_account(ua);
+
+		if (!acc->regint)
```

Comment:
perhaps also check that `acc` is not NULL, in order to avoid a potential null deref crash ...